data/api_v1_to_v2.go

```diff
@@ -120,7 +127,23 @@ func (a *API) getDataV2(ctx context.Context, res *httpResponseWriter) error {
 	// Mongo iterators
 	var iterPumpSettings mongo.StorageIterator
 	var iterUploads mongo.StorageIterator
+	var chanApiCbgs chan []schema.CbgBucket
+	var chanApiBasals chan []schema.BasalBucket
+	var chanApiCbgError, chanApiBasalError chan *detailedError
+	var logErrorDataV2 *detailedError
+	var wg sync.WaitGroup
 
+	var exclusions = excludedTypes()
+	var exclusionList []string
+	groups := 0
+	for key, value := range params.source {
```

Comment:
OpenApi doc doesn't reflect optional parameters for the query (basalBucket and cbgBucket query params)